shared/common-adapters/markdown/generate-emoji-parser.tsx

```diff
@@ -117,15 +117,17 @@ async function buildEmojiFile() {
   const p = path.join(__dirname, 'emoji-gen.tsx')
 
   const {swidth, sheight} = await getSpriteSheetSize()
-  const {emojiLiterals, emojiIndexByName, emojiIndexByChar} = genEmojiData()
-  const regLiterals = emojiLiterals.map((s: string) => escapeRegExp(s).replace(/\\/g, '\\')).join('|')
+  const {emojiIndexByName, emojiIndexByChar} = genEmojiData()
   const regIndex = Object.keys(emojiIndexByName)
     .map((s: string) => escapeRegExp(s).replace(/\\/g, '\\\\'))
     .join('|')
   const data = `/* eslint-disable */
+import emojiRegexNew from 'emoji-regex'
```

Comment:
this has a more compact representation of the emojis vs our naive implementation (that was 200k long). It's smaller and faster. I had code to verify the match was the same and time it across all emojis 1000x and the comparison is
old reg: 4706
new reg: 660